code/modules/cargo/packs/organic.dm

```diff
@@ -184,8 +191,35 @@
 	access = ACCESS_THEATRE
 	crate_type = /obj/structure/closet/crate/secure
 
+/datum/supply_pack/organic/randomized/exoticmeat
+	name = "Meat Crate (Exotic)"
+	desc = "The best cuts in the whole galaxy. Contains 15 assorted exotic meats."
+	cost = 2000
+	contains = list(/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/slime,
+					/obj/item/reagent_containers/food/snacks/meat/slab/killertomato,
+					/obj/item/reagent_containers/food/snacks/meat/slab/bear,
+					/obj/item/reagent_containers/food/snacks/meat/slab/xeno,
+					/obj/item/reagent_containers/food/snacks/meat/slab/spider,
+					/obj/item/reagent_containers/food/snacks/meat/rawbacon,
+					/obj/item/reagent_containers/food/snacks/spiderleg,
+					/obj/item/reagent_containers/food/snacks/carpmeat,
+					/obj/item/reagent_containers/food/snacks/meat/slab/human)
+	crate_name = "exotic meat crate"
+
+/datum/supply_pack/organic/monkeydripmeat
```

Comment:
Now, I know this has been around since #9912 (I was not around to give it a quality check), but this is an easy chemistry trick that can be done without the help of cargo in first place, so please remove.

Reply:
People have been asking for more contraband crates, so I've been trying to add more contraband traits. It only shows up if you hack the console, and people who are curious will order it and try to figure out the gimmick.

Reply:
It's pretty "low quality", simple vague requests do not justify such cheap, meaningless additions. Remove.